napari/_qt/dialogs/activity_dialog.py

```diff
@@ -0,0 +1,217 @@
+from pathlib import Path
+
+from qtpy.QtCore import QPoint, QSize, Qt
+from qtpy.QtGui import QMovie
+from qtpy.QtWidgets import (
+    QDialog,
+    QFrame,
+    QGraphicsOpacityEffect,
+    QHBoxLayout,
+    QLabel,
+    QScrollArea,
+    QSizePolicy,
+    QToolButton,
+    QVBoxLayout,
+    QWidget,
+)
+
+import napari.resources
+
+from ...utils.translations import trans
+from ..widgets.qt_progress_bar import ProgressBar, ProgressBarGroup
+
+
+class ActivityToggleItem(QWidget):
+    """Toggle button for Activity Dialog with progress indicator displayed
+    when there are progress bars present.
+    """
```

Comment:
```suggestion
    """Toggle button for Activity Dialog.
    
    A progress indicator is displayed when there are active progress
    bars.
    """
```

See [PEP-257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)